python/ray/tune/tests/test_tuner_restore.py

```diff
@@ -437,6 +440,63 @@ def test_tuner_restore_latest_available_checkpoint(
     assert result.metrics["iterations_since_restore"] == 5
 
 
+@pytest.mark.parametrize("retry_num", [0, 1])
+def test_retore_retry(retry_num):
```

Comment:
```suggestion
def test_retore_retry(ray_start_4_cpus, retry_num):
```
should shut down ray gracefully

Reply:
yeah I looked at this. Seems like this is a fixture used across board. I wonder why it's running issues now...